### PR TITLE
🆕(content) Add PWA Summit 2022 recap post

### PR DESCRIPTION
**Pull request by Snugug**
Recap post for the event

### CODE REVIEW

File: site/en/posts/pwa-summit-2022.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  - web
+hero:
+  youtube: m-2lLWI_csE
+  alt: PWA Summit 2022, block 1
+authors:
+  - samrichard
+date: 2022-11-04
+---
+
+What sets the web apart from every other development platform? It belongs to all of us. The web’s communal nature is what helps us realize its potential, and no one knows this better than web devs. They enhance the web bit by bit every day to help all of us build better experiences for [more than 5 billion internet users worldwide.](https://www.statista.com/statistics/617136/digital-population-worldwide/) And Progressive Web Apps (PWAs) that make the most of the web’s capabilities while feeling “appy” are a big part of this effort.
```
Comment (MattMonasch) [inline site/en/posts/pwa-summit-2022.md:15]: nit: Isn't starting a sentence with "And" a grammatical no-no?

File: site/en/posts/pwa-summit-2022.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+If you’re storing a lot of data locally, you’re likely familiar with [IndexedDB](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API). Although the API has come a long way, with better cross-browser support and wrapper libraries such as idb and Dexie.js, it still has its drawbacks. For instance, you can’t run more complex queries such as joins and aggregations.
+
+But the web dev community has been finding workarounds. Libraries such as absurd-sql and wa-sqlite let devs run SQL queries on top of data stored in IndexedDB by connecting it to the back end of SQLite.
+
+Another challenge when working with IndexedDB is inadvertent data loss. Browser data might be cleared — either automatically or manually — or simply evicted when disk space is low. The Origin Private File System (an API for high-performance file system access) is a potential solution, but that depends on browser support. To learn more about how client-side data storage is evolving, explore [Jeremy’s talk](https://www.youtube.com/watch?v=tEOzwbvmTpA).
```
Comment (MattMonasch) [inline site/en/posts/pwa-summit-2022.md:47]: Link for Origin Private File System?

File: site/en/posts/pwa-summit-2022.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+## The nuts and bolts of PWAs for practitioners
+
+Getting a handle on the more intricate elements of development will help you take your PWAs to the next level. Explore the talks below to get into the details of storage and real-life use cases.
+
+**[Challenges in storing large amounts of client-side data](https://www.youtube.com/watch?v=tEOzwbvmTpA), by ZenGM Indie Video Game Developer Jeremy Scheff**
+
+If you’re storing a lot of data locally, you’re likely familiar with [IndexedDB](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API). Although the API has come a long way, with better cross-browser support and wrapper libraries such as idb and Dexie.js, it still has its drawbacks. For instance, you can’t run more complex queries such as joins and aggregations.
+
+But the web dev community has been finding workarounds. Libraries such as absurd-sql and wa-sqlite let devs run SQL queries on top of data stored in IndexedDB by connecting it to the back end of SQLite.
```
Comment (MattMonasch) [inline site/en/posts/pwa-summit-2022.md:45]: Library links?

File: site/en/posts/pwa-summit-2022.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+**[Web apps for everyone, everywhere: PWAs to the rescue of people in low-internet areas](https://www.youtube.com/watch?v=3vG6cWW7_n0), by Telos Labs Software Developer Alicia Rojas**
+
+Not everyone has the web at their fingertips at all times, which makes better offline support critical to accessibility. So when Alicia’s team was asked to create an app that farmers in low-connectivity areas could use to troubleshoot issues with their fields, they homed in on the factors critical to offline PWA use: installability and support for offline create, read, update, and delete (CRUD) actions.
+
+The Background Sync API — which lets web apps defer actions until there’s stable network connection — and IndexedDB were critical to supporting these CRUD actions. Let’s say a farmer tried to create a new diagnosis survey without the internet. IndexedDB, a local database, would store this request as a JSON object. Then, once the farmer was back online, Background Sync would detect this connection to synchronize with the server. To learn more about what web apps can offer and how you can enhance offline capabilities, check out [Alicia’s talk](https://www.youtube.com/watch?v=3vG6cWW7_n0).
```
Comment (MattMonasch) [inline site/en/posts/pwa-summit-2022.md:53]: Background Sync API link?

File: site/en/posts/pwa-summit-2022.md (hunk 1 above)
Comment (Snugug) [reply site/en/posts/pwa-summit-2022.md:15]: I'd generally agree here, but this copy is shared between the companies that put the event on and is locked so I don't want to go rewording stuff like this, if that's OK.

Comment (MattMonasch) [reply site/en/posts/pwa-summit-2022.md:15]: Yeah, no worries then.

**Pull request merged**